graphium/trainer/predictor.py

```diff
@@ -290,10 +297,16 @@ def compute_loss(
 
         if weights is not None:
             raise NotImplementedError("Weights are no longer supported in the loss")
-        all_task_losses = {
-            task: wrapped(preds=preds[task], target=targets[task])
-            for task, wrapped in wrapped_loss_fun_dict.items()
-        }
+
+        # Get the loss for each task
+        all_task_losses = {}
+        for task, wrapped in wrapped_loss_fun_dict.items():
+            this_loss = wrapped(preds=preds[task], target=targets[task])
+            if loss_empty_factor is not None:
+                this_loss = this_loss * loss_empty_factor[task]
+            all_task_losses[task] = this_loss
+
+        # Sum the losses
         total_loss = torch.sum(torch.stack(list(all_task_losses.values())), dim=0)
```

Comment:
I think here it is stacking lists of losses:
task1: [0, 1, 2]
task2: [1.5, 2, 3.3]
task3: [0.3, 4,3, 0]
stacked: [1.8, 7, 5.3] then apply torch,sum --> 1.8 + 7 + 5.3 = 14.1
let me know if this makes sense.